### ISSUE TITLE
Please show the battery life for my headset device

### ISSUE THREAD

**Issue by Greenbear59** [blueman-project/blueman#1453]
blueman: 2.1.2
BlueZ: 5.53-0ubuntu3
Distribution: Ubuntu 20.04.1
Desktop environment: Xfce

* [x] I have consulted the [Troubleshooting page](https://github.com/blueman-project/blueman/wiki/Troubleshooting) and done my best effort to follow.

When i connect my headset (freeVoice Space FBT650BTS) to my mobile it shows the percentage of power (ie. the battery life).
⇒ please add this feature to the blueman

<!--

ℹ Please use some service like [Pastebin](http://pastebin.com/) or [GitHub Gist](https://gist.github.com/) to post your logs and keep the thread clean and readable. Make sure to describe the exact steps you took when producing it.

->

**Comment by cschramm** [comment]
Duplicate of #947, solved in #1417 and #1418 is already available (will be in version 2.2; release mainly depends on #1414).
